charts/airbyte-temporal/README.md

```diff
@@ -27,6 +27,7 @@ Helm chart to deploy airbyte-temporal
 | global.database.port | string | `"5432"` |  |
 | global.database.secretValue | string | `"postgresql-password"` |  |
 | global.deploymentMode | string | `"oss"` |  |
+| global.extraContainers | list | `[]` |  |
```

Comment:
we are removing temporal from the Airbyte Cloud deployment, so we won't need to wrap this with extra containers. we can still keep this for consistency.